crates/topos-node/src/main.rs

```diff
@@ -0,0 +1,4 @@
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
dumb question maybe: why do we need a `main` here?

Reply:
The goal is to be able to launch `topos-node` binary directly

Reply:
So this is a placeholder of sorts then?

Reply:
Yes we can say that it's a placeholder